spec/lib/listen/directory_spec.rb

```diff
@@ -86,6 +86,20 @@
         end
       end
 
+      context 'when network share is disconnected' do
+        before { allow(dir).to receive(:children) { fail Errno::EHOSTDOWN} }
+
+        it 'reports no changes' do
+          expect(queue).to_not receive(:change)
+          described_class.scan(queue, record, dir, '.', options)
+        end
+
+        it 'unsets record dir path' do
+          expect(async_record).to receive(:unset_path).with(dir, '.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.